cmd/csi-node-driver-registrar/main.go

```diff
@@ -51,7 +53,13 @@ var (
 	healthzPort             = flag.Int("health-port", 0, "(deprecated) TCP port for healthz requests. Set to 0 to disable the healthz server. Only one of `--health-port` and `--http-endpoint` can be set.")
 	httpEndpoint            = flag.String("http-endpoint", "", "The TCP network address where the HTTP server for diagnostics, including the health check indicating whether the registration socket exists, will listen (example: `:8080`). The default is empty string, which means the server is disabled. Only one of `--health-port` and `--http-endpoint` can be set.")
 	showVersion             = flag.Bool("version", false, "Show version.")
-	version                 = "unknown"
+
+	// kubelet registration succeeded flags
+	kubeletRegistrationSucceededLockfilePath = flag.String("kubelet-registration-succeeded-lockfile-path", "", "If set, a temp file with this name will be created after the kubelet registration process succeeds.")
+	kubeletRegistrationSucceededMode         = flag.String("kubelet-registration-succeeded-mode", "", `Used only when --kubelet-registration-succeeded-lockfile-path is set to check if the kubelet registration process succeeded in an exec probe, the only posible value is "probe".`)
```

Comment:
I would probably name this something more generic like "--mode=kubelet-registration-probe". That would let us extend node-driver-registrar to potentially incorporate other health checks like https://github.com/kubernetes-csi/node-driver-registrar/pull/99

Reply:
perfect, I added a running mode and I'm also reusing `kubelet-registration-path` in the exec probe, I also added an example in the README.